web/libs/editor/src/components/VideoCanvas/VideoCanvas.tsx

```diff
@@ -167,7 +171,9 @@ export const VideoCanvas = memo(
         if (!contextRef.current) return;
 
         const currentTime = videoRef.current?.currentTime ?? 0;
-        const frameNumber = Math.round(currentTime * framerate);
+        const frameNumber = isFF(FF_VIDEO_FRAME_SEEK_PRECISION)
+          ? Math.floor(currentTime * framerate) + 1
```

Comment:
this should be the same
```suggestion   
          ? Math.ceil(currentTime * framerate)
```

Reply:
Thinking about the overall solution and what is semantically be achieved it does make more sense. We are always looking for the **next** frame of the browser precision rounded time.